Gems/ScriptCanvas/Code/Include/ScriptCanvas/AutoGen/ScriptCanvasGrammarRegistry_Source.jinja

```diff
@@ -87,7 +87,9 @@ namespace ScriptCanvas
 
     AZStd::vector<AZ::ComponentDescriptor*> {{autogenTargetName}}GrammarRegistry::GetComponentDescriptors()
     {
-        return AZStd::vector<AZ::ComponentDescriptor*>({
+        if (m_cachedDescriptors.empty())
+        {
+            AZStd::vector<AZ::ComponentDescriptor*> descriptors({
```

Comment:
instead of creating a temporary variable, you should be able to do something like
```
m_cachedDescriptors = {
...
};
```

Reply:
I changed it to move from the temporary into the cached member, but I'll try this out.

Reply:
Done